src/netex_france/stops.rs

```diff
@@ -283,7 +297,10 @@ impl<'a> StopExporter<'a> {
         })
     }
 
-    fn generate_centroid(&self, coord: &'a Coord) -> Result<Element> {
+    fn generate_centroid(&self, coord: &'a Coord) -> Result<Option<Element>> {
+        if coord.lat == 0.0 && coord.lon == 0.0 {
```

Comment:
```suggestion
       if *coord == Coord::default() {
```

Reply:
Done in 86a310621db97e5c506bc77a9c545e4559a867f9.